pages/api/bib/[id].ts

```diff
@@ -0,0 +1,41 @@
+import type { NextApiRequest, NextApiResponse } from "next"
+import { fetchBib } from "./index"
+import { PATHS } from "../../../src/config/constants"
+import { mapQueryToBibParams } from "../../../src/utils/bibUtils"
+
+/**
+ * Default API route handler for Bib page
+ *
+ * It handles a get request for a given bib id by internally calling fetchBib
+ * and handling the response based on the status returned there.
+ *
+ * Note: This is primarily used for debugging since the Bib page fetches the
+ * result by calling fetchBib directly in getServerSideProps.
+ */
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const id = Array.isArray(req.query.id) ? req.query.id[0] : req.query.id
```

Comment:
In what scenario can `id` potentially be an array?

Reply:
Good point. Any query param is allowed to be an array, but the ID will never be an array in our usage. I cast it as a string instead.